packages/clerk-js/src/ui/types.ts

```diff
@@ -50,7 +54,8 @@ type ComponentMode = 'modal' | 'mounted';
 export type SignInCtx = SignInProps & {
```

Comment:
should we update `SignInProps` to accept `SignInForceRedirectUrl` instead of only adding it to the context type?

Reply:
My intent was to keep the prop types unchanged and handle it internally, hence only applying it to the context type here.

Reply:
ahhh right right